docs/howto/openstack/nova/config-drive.md

```diff
@@ -0,0 +1,95 @@
+# Store metadata on a configuration drive
+
+Configuration drive is a read only drive that is attached to an 
+instance during boot. The instance can then mount the drive and
+read files from it. Configuration drives are used as a data source for cloud-init
+and also for passing data to your instances to configure.
+
+# Enable the configuration drive
+
+To enable the configuration drive, you need to pass the parameter `--config-drive`
+with `true` flag to the `openstack server create` command and can also
+specify the user data you want to inject into the server.
+
+## Example shell script
+
+A bash script is a common way to inject user data to an instance.
+The following script will install `tree` command during instance
```

Comment:
It's "*the* `tree` command". But `apt` does not install a command; it installs a package.

Reply:
Done